client/syntaxes/aspnetcorerazor.tmLanguage.json

```diff
@@ -3,7 +3,345 @@
 	"scopeName": "text.aspnetcorerazor",
 	"patterns": [
 		{
-			"include": "text.html.cshtml"
+			"include": "#razor-directives"
```

Comment:
Where was all this copied from?

Reply:
Included you in an email. It was copied from https://github.com/microsoft/vscode/blob/673295580adb8fc4ebd70dcc38113e61ad491dba/extensions/razor/syntaxes/cshtml.tmLanguage.json

Reply:
I see so that file can be deleted? Or is it still used somewhere?

Reply:
Not sure about VSCode's policies with enabling coloring even without extensions installed so the real answer is "maybe" it can be deleted? We'll have to follow up with them to decide on what to do.